ansible/roles/ui/templates/nginx.conf.j2

```diff
@@ -62,7 +62,7 @@ http {
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
             proxy_set_header X-Forwarded_Proto $scheme;
 
-            proxy_pass http://{{ donkey.host }}:{{ kifshare.port }}/;
+            proxy_pass http://{{ terrain.host }}:{{ kifshare.port }}/;
```

Comment:
Yeah, that should probably be `kifshare.host`. I'll try to commit a fix for that today.